lib/workers/repository/update/pr/changelog/index.ts

```diff
@@ -30,10 +31,10 @@ export async function getChangeLogJSON(
 
     switch (platform) {
       case 'gitlab':
-        res = await sourceGitlab.getChangeLogJSON(config);
+        res = await sourceGitlab.getChangeLogJSON(config, sourceUrl);
```

Comment:
```suggestion
        res = await sourceGitlab.getChangeLogJSON({ ...config, sourceUrl });
```

This is just a suggestion, I'm not sure if it's better or not but I think it would mean we don't need to change the other files, less chance of error.

Reply:
this was my first approach so to reduce PR diff, but this doesnt matter in this respect.

the change is quite small regardless